subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/java/JavaFirstUsePerformanceTest.groovy

```diff
@@ -30,7 +30,6 @@ class JavaFirstUsePerformanceTest extends AbstractCrossVersionPerformanceTest {
         runner.testProject = testProject
         runner.gradleOpts = ["-Xms${testProject.daemonMemory}", "-Xmx${testProject.daemonMemory}"]
         runner.tasksToRun = ['tasks']
-        runner.args = ['--recompile-scripts'] // This is an approximation of first use: we recompile the scripts
```

Comment:
If we just remove this, then this test no longer makes any sense. We need to add an experiment listener that deletes the script cache after the build.

Reply:
Given that we're deprecating things instead of removing, I keep this flag unchanged.